tests/test_integration/test_eval_loop_integration.py

```
@@ -945,3 +945,47 @@ def test_config_changed_resume_clears_leftover_sidecars(self, tmp_path):
 
         assert not (log_dir / "trajectory_overflow").exists()
         assert not (log_dir / "capture_markers").exists()
+
+
+class _UnscorableEnv(EvalEnvironment):
+    """Second of three problems cannot be scored at all (issue #1140)."""
+
+    name = "mock_unscorable"
+
+    def __init__(self):
```

Ruff (0.16.3)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)